app/controllers/actions/agent/AgentIdentifierAction.scala

```diff
@@ -61,6 +61,9 @@ class AuthenticatedAgentIdentifierAction @Inject() (
             case None =>
               logger.debug(s"AgentIdentifierAction: Agent without HMRC-AS-AGENT enrolment. Enrolments: $enrolments")
               Future.successful(Redirect(controllers.agent.routes.AgentUseAgentServicesController.onPageLoad))
+            case _ =>
+              logger.debug(s"AgentIdentifierAction: Agent without enrolment. Enrolments: $enrolments")
+              Future.successful(Redirect(controllers.agent.routes.AgentUseAgentServicesController.onPageLoad))
```

Comment:
In this case I don't think a `case _ `should be added, we should just change the `case None` above into a `case _`